src/stories/Forms/Select/Select.tsx

```diff
@@ -281,7 +301,7 @@ export interface SelectProps extends GlobalInputProps {
    * Option components to be rendered.
    */
   children: React.ReactNode;
-  showItems?: number;
+  multiselect?: boolean;
```

Comment:
I think it is very cool that just one prop changes this select to a multiselect!